File: tests/src/main/java/com/github/klee0kai/test/house/rooms/BedRoom.java
package com.github.klee0kai.test.house.rooms;

import com.github.klee0kai.test.house.kitchen.storagearea.StoreArea;

import java.util.UUID;

public class BedRoom {

    public UUID uuid = UUID.randomUUID();

    public final StoreArea storeArea;


    public BedRoom(StoreArea storeArea) {
        this.storeArea = storeArea;
    }
}

File: tests/src/commonMain/kotlin/com/github/klee0kai/test/house/rooms/BedRoom.kt
@file:OptIn(ExperimentalUuidApi::class)

package com.github.klee0kai.test.house.rooms

import com.github.klee0kai.test.house.kitchen.storagearea.StoreArea
import kotlin.uuid.ExperimentalUuidApi
import kotlin.uuid.Uuid

class BedRoom(val storeArea: StoreArea?) {
    var uuid: String = Uuid.random().toString()
}
